### PR TITLE
df+du+ls: Add an option to print the human readable sizes in powers of 10

### PR DESCRIPTION
**Pull request by kuzux**
Such options existed in linux utilities of the same names.

### CODE REVIEW

**Review by supercomputer7 — APPROVED** [review]
Seems OK to me, as long as you tested it and it looked OK :)

**Comment by GMTA** [comment]
This is not right - only `df` works with `-H` as an argument, but `du` and `ls` dereference symbolic links when you pass them that short option. Those last two tools use the long opt `--si` only.

File: Userland/Utilities/du.cpp [hunk 1]
```diff
@@ -97,6 +98,7 @@ ErrorOr<void> parse_args(Main::Arguments arguments, Vector<DeprecatedString>& fi
     args_parser.add_option(du_option.all, "Write counts for all files, not just directories", "all", 'a');
     args_parser.add_option(du_option.apparent_size, "Print apparent sizes, rather than disk usage", "apparent-size", 0);
     args_parser.add_option(du_option.human_readable, "Print human-readable sizes", "human-readable", 'h');
+    args_parser.add_option(du_option.human_readable_si, "Print human-readable sizes in SI units", "si", 'H');
```
Comment (GMTA) [inline Userland/Utilities/du.cpp:101]: `du` does not use `-H` for this; please remove

File: Userland/Utilities/ls.cpp [hunk 2]
```diff
@@ -113,6 +114,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     args_parser.add_option(flag_print_numeric, "In long format, display numeric UID/GID", "numeric-uid-gid", 'n');
     args_parser.add_option(flag_hide_group, "In long format, do not show group information", nullptr, 'o');
     args_parser.add_option(flag_human_readable, "Print human-readable sizes", "human-readable", 'h');
+    args_parser.add_option(flag_human_readable_si, "Print human-readable sizes in SI units", "si", 'H');
```
Comment (GMTA) [inline Userland/Utilities/ls.cpp:117]: `ls` does not use `-H` for this; please remove and leave `--si`

**Review by GMTA — CHANGES_REQUESTED** [review]
Two `-H` short options should be removed.

**Review by GMTA — APPROVED** [review]
LGTM!

**Pull request merged**